src/isPalindrome.js

```diff
@@ -19,7 +19,12 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  if (str === '') {
+    return true;
+  }
+  const letters = str.toUpperCase().match(/\w/gi).join('');
+  const letters2 = str.toUpperCase().match(/\w/gi).reverse('').join('');
```

Comment:
you can use here `letters`
```suggestion
  const letters2 = letters.split('').reverse('').join('');
```

Reply:
`letters2` bad name, rename this variable